Installing-and-Configuring/OS/Generic.md

```diff
@@ -0,0 +1,258 @@
+---
+layout: default
+title: Generic
+permalink: /OS/Generic/
+parent: Operating Systems
+grand_parent: Installing and configuring
+nav_order: 1
+---
+
+<!-- markdownlint-disable MD033 -->
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+<!-- markdownlint-enable MD033 -->
+
+Generic OS Installation
+===
+
+Insert OS installation media into one of the USB3 ports (blue on Thinkpads).
+ Turn the system on and navigate to the USB boot in the graphical heads menu. 
+
+Main Menu => Boot Options => USB Boot
+
+From here you will be presented with a menu of the boot options on the USB 
+ media.  Choose the one you wish to use and Press enter.
+
+
+Securely Booting Installation Media
+----
+[For certian OSes](https://github.com/osresearch/heads/tree/master/initrd/etc/distro/keys),
+ the Heads boot process supports standard OS bootable media where the USB
+ drive contains the installation media.  This is generally created using `dd` 
+ or `unetbootin` etc.  It is your responsibility to verify the integrity of the
+ media before using it.  Check your ISO downloads against the signatures 
+ provided by vendors.
+ 
+### Alternative Options
```

Comment:
WIP  I reworded a little based on what I think you are saying but please review. I'm not sure I fully grasp all of the aspects on this.  Will push in a few minutes with everything else on this PR.

Reply:
I reworked this again after review.  I think I see how I was confused.  I also moved the content up to the index page rather than in the Generic OS page.  There was duplicate content on both pages.